roles/vm_image/examples/example2.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Create volumes 2
+  hosts: localhost
+  module_defaults:
```

Comment:
The examples are really similar. What do we want to show in both? Do you think it would be good idea to have only one example?

Reply:
Hmm, the only reasoning for the second file was to showcase the `kubevirt_cdi_upload` upload host override. I'll drop the other examples from that file and just leave the upload one.

Reply:
Done.